bindata/network/ovn-kubernetes/common/alert-rules.yaml

```diff
@@ -62,3 +62,12 @@ spec:
       for: 15m
       labels:
         severity: warning
+    - alert: OVNKubernetesResourceRetryFailure
+      annotations:
+        summary: OVN Kubernetes experiencing Kubernetes Resource Retry Failure at an elevated rate.
+        description: OVN Kubernetes experiencing Kubernetes resource processing failure even
+         after reaching its maximum retry limit and was no longer processed.
+      expr: rate(ovnkube_resource_retry_failures_total[1m]) > 0
+      for: 10m
```

Comment:
yes, you're right. now changed the expr with `changes(ovnkube_resource_retry_failures[24h]) > 0`, hope this would give a day alert if any increase is seen on retry failure counter. I've not tested it myself, will test and get back on this.